src/components/course/data/constants.js

```diff
@@ -3,6 +3,8 @@ import GetSmarterLogo from '../../../assets/icons/getsmarter-header-icon.svg';
 export const COURSE_PACING_MAP = {
   SELF_PACED: 'self_paced',
   INSTRUCTOR_PACED: 'instructor_paced',
+  INSTRUCTOR: 'instructor',
+  SELF: 'self',
```

Comment:
nit: As discussed on Friday (I think?), let's at least include a comment to explain the `instructor` and `self` are values coming from `enterprise_course_enrollments` API. That way, future us (at least, for myself haha), aren't potentially confused about the nuance for why there are distinct keys.